### ISSUE TITLE
[Bug]: gcli ignores `api-base` in configuration

### ISSUE THREAD

**Issue by jiriks74** [herrhotzenplotz/gcli#138]
As per the man page

```
       forge-type  The type of the forge. May be one of:
                   •   github
                   •   gitlab
                   •   gitea

       api-base    (optional) Used to override the API base URL of the forge. This is useful for self-hosted instances.  Depending on the “forge-type” the default values are:

                   forge-type    default value
                   github        https://api.github.com
                   gitlab        https://gitlab.com/api/v4
                   gitea         https://codeberg.org/api/v1
```

I set values for my account:

```
jiriks74 {
  forge-type = gitea
  api-base = https://gitea.stefka.eu/api/v1
  account = jiriks74
  token = xyz
}
```

But the tool still reaches for Codeberg:

```
❯ gcli -a jiriks74 pulls
gcli: error: could not fetch pull requests: request to https://codeberg.org/api/v1/repos/jiriks74/os%2Ehome%2Enix/pulls?state=open failed with code 404: API error: GetUserByName
```

**Comment by herrhotzenplotz** [comment]
Sorry, this is a documentation/typo bug... I am aware of this but forgot about this. Thank for reminding me!

the fix is to say `apibase` instead of `api-base`. I will add an alias in the future to reduce breakage.

**Comment by herrhotzenplotz** [comment]
@jiriks74 I pushed two changes to `fix/bad-api-base-thing`. The changes should both fix the buggy behaviour to be as it is documented in the manual page and also remove some other references to `apibase` (without dash). If that works for you I'll apply the changes to trunk.

**Comment by jiriks74** [comment]
I've figured out, by searching through issues in this repo, that `apibase` works..

I'll look into this branch later as I'm not feeling like rebuilding another package now. I'll update you when I'll do so.

**Comment by jiriks74** [comment]
Also, thx for the quick response.

**Comment by herrhotzenplotz** [comment]
Oops, my original fix had a silly bug in it, it's fixed now: 8839f85
